packages/ckeditor5-dev-changelog/src/utils/normalizeentry.ts

```diff
@@ -3,33 +3,24 @@
  * For licensing, see LICENSE.md.
  */
 
-import type { ParsedFile } from '../types.js';
-
-export function normalizeEntry( entry: ParsedFile, singlePackage: boolean ): ParsedFile {
-	// Normalize type.
-	const typeCapitalized = capitalize( entry.data.type );
-	const typeNormalized = typeCapitalized === 'Fixes' ? 'Fix' : typeCapitalized;
-
-	// Normalize breaking-change.
-	const breakingChange = entry.data[ 'breaking-change' ];
-	const breakingChangeLowerCase = String( breakingChange ).toLowerCase();
-	let breakingChangeNormalized: 'minor' | 'major' | boolean | undefined;
-
-	if ( singlePackage ) {
-		if ( breakingChangeLowerCase === 'true' ) {
-			breakingChangeNormalized = true;
-		} else if ( breakingChangeLowerCase === 'minor' ) {
-			breakingChangeNormalized = 'minor';
-		} else if ( breakingChangeLowerCase === 'major' ) {
-			breakingChangeNormalized = 'major';
+import { TYPES } from '../constants.js';
+import type { NormalizedType, ParsedFile } from '../types.js';
+
+export function normalizeEntry( entry: ParsedFile ): ParsedFile {
+	// Normalize Type.
+	const typeCapitalized = capitalize( entry.data.type ) as NormalizedType;
```

Comment:
This type should not be cast to `NormalizedType`, because user can provide an invalid value, e.x. `Fixed`.